vegaprotocol/vega
# Pick up the wallet changes when the service is started

Close #5987 

## How to test?

1. Start the service `vegawallet service run` (**don't shut it down until the end**)
2. Create the wallet `vegawallet create`
3. Authenticate using the created wallet with `client.connect_wallet`
4. Call `client.list_keys`, and assess the keys (you will have to update the permissions)
5. Generate new keys on the wallet you are connected with, using the CLI `vegawallet key generate`
6. Call `client.list_keys`, again, and verify the keys are updated (if not, verify the permission doesn't have specific allowed keys)
7. Delete the wallet using the CLI `vegawallet delete`
8. Verify the connection has been killed by calling `client.list_keys`
9. Recreate the wallet `vegawallet create`
10. Re-authenticate with `client.connect_wallet`
11. Call `client.list_keys`, and assess the key (you will have to update the permissions)
12. Change the passphrase of the wallet `vegawallet passphrase update`
13. Verify the connection has been killed by calling `client.list_keys`
